src/Tools/NodeDocumentationMarkdownGenerator/MarkdownHandler.cs

```diff
@@ -6,7 +6,7 @@
 using System.Reflection;
 using System.Text;
 using System.Text.RegularExpressions;
-using Dynamo.Wpf.Interfaces;
+using Dynamo.Interfaces;
```

Comment:
Does the code not compile any longer without these additional `using`s or are these required at runtime. Even though the type forwarding includes the complete namespace, is this still required?